module/cmd/gravity/cmd/gentx.go

```diff
@@ -149,7 +150,7 @@ $ %s gentx my-key-name 1000000stake 0x033030FEeBd93E3178487c35A9c8cA80874353C9 c
 			}
 
 			// validate orchestrator account in genesis and warn if not found
-			if err = genutil.ValidateAccountInGenesis(genesisState, genBalIterator, orchAddress, coins, cdc); err != nil {
+			if err = validateAccountPresentInGenesis(genesisState, genAccountIterator, orchAddress, coins, cdc); err != nil {
 				cmd.PrintErrf("orchestrator address not found in genesis file: %s\n", orchAddress)
 				return nil
```

Comment:
So looks like this will still error out if the address isn't in genesis. I think you would want to just `cmd.PrintErrf` and not have the return like so:

```suggestion
```

Reply:
I think we want to have it error in if the address isn't in genesis but not require the address to have a balance.